game-core/src/main/java/games/strategy/debug/error/reporting/StackTraceErrorReportFormatter.java

```diff
@@ -94,14 +95,14 @@ private static String throwableToString(final Throwable e) {
         + Optional.ofNullable(e.getStackTrace())
             .map(
                 trace -> "## Stack Trace\n"
-                    + String.join("\n", stackTraceToString(trace))
+                    + stackTraceToString(trace)
                     + "\n\n")
             .orElse("");
   }
 
-  private static String[] stackTraceToString(final StackTraceElement[] stackTrace) {
+  private static String stackTraceToString(final StackTraceElement[] stackTrace) {
     return Arrays.stream(stackTrace)
         .map(StackTraceElement::toString)
-        .toArray(String[]::new);
+        .collect(Collectors.joining("\n\n"));
```

Comment:
I'm curious where the additional newline comes from.

Reply:
Good eye, fixed: ed10bfa